src/pages/ReportPage.vue

```diff
@@ -3,73 +3,338 @@
+                </div>
+              </div>
+            </q-card-section>
+          </q-card>
+        </div>
+      </template>
+    </div>
+    <div class="row q-col-gutter-md q-pa-md">
+      <div class="col-6">
+        <Radar :data="dataRadar" :options="options" />
+      </div>
+      <div class="col-6">
+        <div class="row">
+          <p class="text-weight-bold">CVSS Global Score</p>
+        </div>
+        <div class="row q-col-gutter-md q-mb-md">
+          <div class="col-6">
+            <q-card>
+              <q-card-section class="text-center text-weight-bold"> Actual </q-card-section>
+              <q-card-section class="text-center">
+                CVSS : {{ detailInitialResponse.global_cvss_score }}
+              </q-card-section>
+              <q-card-section class="text-center">
+                Total Vulnerabilities : {{ detailInitialResponse.global_total_vulnerabilities }}
+              </q-card-section>
+            </q-card>
+          </div>
+          <div
+            class="col-6"
+            v-if="
+              totalUpdated.expected_global_cvss_score !== 0 &&
+              totalUpdated.expected_global_total_vulnerabilities !== 0
+            "
+          >
+            <q-card>
+              <q-card-section class="text-center text-weight-bold"> Anticipated </q-card-section>
+              <q-card-section class="text-center">
+                CVSS : {{ totalUpdated.expected_global_cvss_score }}
+              </q-card-section>
+              <q-card-section class="text-center">
+                Total Vulnerabilities : {{ totalUpdated.expected_global_total_vulnerabilities }}
+              </q-card-section>
+            </q-card>
+          </div>
+        </div>
+
+        <q-card v-if="showData">
+          <q-card-actions align="right">
+            <q-btn icon="close" flat dense @click="showData = false"></q-btn>
+          </q-card-actions>
+          <q-card-section>
+            <div>
+              {{ vectorData.name }}
+            </div>
+            <div><q-slider v-model="currentCvss" marker-labels :min="0" :max="10" disable /></div>
+            <div>
+              <span>Type : </span><span>{{ vectorData.type }}</span>
+            </div>
+            <div>
+              <span>Severity : </span><span>{{ vectorData.severity }}</span>
+            </div>
+            <div>
+              <span>Description : </span><span>{{ vectorData.description }}</span>
+            </div>
+            <div>
+              <span>Privileges Required : </span><span>{{ vectorData.privileges_required }}</span>
+            </div>
+            <div>
+              <span>Classification : </span><span>{{ vectorData.classification }}</span>
+            </div>
+            <div>
+              <span>Integrity : </span><span>{{ vectorData.integrity }}</span>
+            </div>
+            <div>
+              <span>Availability : </span><span>{{ vectorData.availability }}</span>
```

Comment:
These three titles would be better off as:

```suggestion
              <span>Attack Vector Classification : </span><span>{{ vectorData.classification }}</span>
            </div>
            <div>
              <span>Integrity Impact : </span><span>{{ vectorData.integrity }}</span>
            </div>
            <div>
              <span>Availability Impact : </span><span>{{ vectorData.availability }}</span>
```